native/explorer/src/dataframe.rs

```diff
@@ -413,6 +413,21 @@ pub fn df_select_at_idx(data: ExDataFrame, idx: usize) -> Result<Option<ExSeries
     })
 }
 
+#[rustler::nif(schedule = "DirtyCpu")]
+pub fn df_select_at_range(
+    data: ExDataFrame,
+    range: Vec<usize>,
+) -> Result<ExDataFrame, ExplorerError> {
+    df_read!(data, df, {
+        let columns = range
+            .into_iter()
+            .map(|i| df.select_at_idx(i).unwrap().clone())
```

Comment:
If you use a `Range`, maybe you can select directly with [`select_by_range`](https://pola-rs.github.io/polars/polars/frame/struct.DataFrame.html#method.select_by_range). I'm not sure if will be faster or use less memory.

Another idea is to use [`get_column_names`](https://pola-rs.github.io/polars/polars/frame/struct.DataFrame.html#method.get_column_names) and select them by the indexes (the  `range` you have today), and after that use [`select`](https://pola-rs.github.io/polars/polars/frame/struct.DataFrame.html#method.select).

Reply:
The problem with using `select_at_range` is that it would break the functionality of selecting with a list of values.

Reply:
I see, makes sense. My idea was to avoid selecting each series individually. But I'm not sure if this is a problem. Probably not :)

Reply:
Another idea is to solve the problem entirely in Elixir. You can select the column names based in their position and pass down the names you want to select (in the order of the range or column idxs). This can have the benefit to avoid the need of adding another NIF function. WDYT?

Reply:
@philss I like that idea a lot. I'm not sure how all of this is going to play into lazy as there isn't a set column index in a lazy frame so it seems like leaning on the names is the best bet.

Reply:
It's a great idea. I'll make the necessary changes!